packages/sanity/src/core/preview/utils/getPreviewPaths.ts

```diff
@@ -1,11 +1,18 @@
 import {SchemaType} from '@sanity/types'
 import {PreviewPath} from '../types'
 
+const DEFAULT_PREVIEW_PATHS: PreviewPath[] = [['_createdAt'], ['_updatedAt']]
+
 /** @internal */
 export function getPreviewPaths(preview: SchemaType['preview']): PreviewPath[] | undefined {
   const selection = preview?.select
 
   if (!selection) return undefined
 
-  return Object.values(selection).map((value) => String(value).split('.'))
+  // Transform the selection dot-notation paths into array paths.
+  // Example: ['object.title', 'name'] => [['object', 'title'], ['name']]
+  const paths = Object.keys(selection).map((value) => String(value).split('.')) || []
+
+  // Return the paths with the default preview paths appended.
+  return paths.concat(DEFAULT_PREVIEW_PATHS)
```

Comment:
This will add `_createdAt` and `_updatedAt` to objects types as well as documents. It's not a huge deal (I think), but would be preferable if we could add them only for document types.

Reply:
I see and I've been looking into this - but I'm wondering if it's actually necessary, as previews will only be created for documents and not for objects, no?

Reply:
Ah, you're absolutely right. This happens right before fetching (I initially thought this happened at the "generate default preview config" phase), and we never have to fetch anything when previewing objects. Thanks for clearing that up!